Sources/SwifterSwift/AppKit/NSViewExtensions.swift

```diff
@@ -6,7 +6,13 @@ import AppKit
 // MARK: - Properties
 
 public extension NSView {
-    /// SwifterSwift: Border color of view; also inspectable from Storyboard.
+    /// SwifterSwift: Border color of the view; also inspectable from Storyboard.
+    ///
+    ///    ```swift
+    ///    view.borderColor = NSColor.red
```

Comment:
Can't you just do this?
```suggestion
    ///    view.borderColor = .red
```